PrivateHeaders/APPX/ZIP.h

```diff
@@ -222,21 +262,28 @@ namespace appx {
         sink.Write(sizeof(data), data);
     }
 
+    // this writes [Content_Types].xml
     template <typename TSink>
     ZIPFileEntry WriteContentTypesZIPFileEntry(
-        TSink &sink, off_t offset,
+        TSink &sink, off_t offset, bool isBundle,
         const std::vector<ZIPFileEntry> &otherEntries)
     {
         // we only need the filenames from otherEntries
         // [Content_Types].xml contains the ZIP-escaped
         // names, hence the use of sanitizedFileName
-        static const std::unordered_map<std::string, const char *>
+        static std::unordered_map<std::string, const char *>
             sKnownContentTypes = {
+                {"appx", "application/vnd.ms-appx"},
                 {"dll", "application/x-msdownload"},
                 {"exe", "application/x-msdownload"},
                 {"png", "image/png"},
-                {"xml", "application/vnd.ms-appx.manifest+xml"},
             };
+        if (isBundle) {
+            sKnownContentTypes.insert({{"xml", "application/vnd.ms-appx.bundlemanifest+xml"}});
```

Comment:
`emplace` should be preferred over `insert`. It's sometimes more efficient (not likely in this case) and requires less syntax noise.